incubator/esbuild-bundle-analyzer/src/webpackStats.ts

```diff
@@ -40,39 +45,70 @@ function getLine(filePath: string, keyword: string): string {
 }
 
 /**
- * Returns a simpler and more readable path which starts from node_modules, e.g.:
- * ../../.store/@test-library@0.0.1-b55dbe3d1aed7a6c074d/node_modules/@test-library/a/b/test.js
+ * Returns a cleaner userRequest by removing the src/index.ts(x) and lib/index.js(x)
+ * from the import path.
+ */
+export function getCleanUserRequest(userRequest?: string): string {
+  if (!userRequest) return "";
+
+  const patternToRemoveRegex =
+    /\/(src|lib|dist|build)\/index(\.[^.]+)?(\.[^.]+)?$/;
+
+  const match = userRequest.match(patternToRemoveRegex);
+  if (match) {
+    return userRequest.replace(match[0], "");
+  }
+
+  return userRequest;
```

Comment:
I think you can simplify this:
```suggestion
  return userRequest.replace(patternToRemoveRegex, "");
```